.github/workflows/packer.yml

```diff
@@ -1,40 +0,0 @@
-name: Packer
-on:
-  push:
-    branches:
-      - main
-    paths:
-      - 'infrastructure/ansible/**'
-      - 'infrastructure/packer/**'
-
-jobs:
-  packer:
-    runs-on: ubuntu-latest
-    name: packer
-
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v2
-
-      # # fix backwards incompatibilities in template
-      # - name: Fix Template
-      #   uses: hashicorp/packer-github-actions@master
-      #   with:
-      #     command: fix
-
-      # Validate templates
-      - name: Validate Template
-        uses: hashicorp/packer-github-actions@master
-        with:
-          command: validate
-          target: infrastructure/packer
-
-      # Build artifact
-      - name: Build Artifact
-        uses: hashicorp/packer-github-actions@master
-        with:
-          command: build
-          arguments: "-color=false -on-error=abort"
-          target: infrastructure/packer
-        env:
-          PACKER_LOG: 1
```

Comment:
@AlmightyYakob what is the purpose of this action? Currently it always throws errors, as do the rest of the steps that use the `hashicorp/packer-github-actions` Action. I've rewritten the others to use the packer CLI directly, should I do the same for this one?

Reply:
hashicorp/packer-github-actions seems to have an upstream bug https://github.com/hashicorp/packer-github-actions/issues/20

Reply:
I think this is fine. Honestly this feels more natural anyway.

Reply:
To answer your question: The reason for running `build` under certain conditions is to automatically rebuild our base node AMI whenever needed. So if we added some system dependency that we wanted running on all nodes, the AMI would get rebuilt and it would become present.

Reply:
So we probably still want this packer CI script, but just using the packer CLI instead of the upstream github action.